package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "types": "./dist/index.d.ts",
   "module": "dist/index.js",
-  "homepage": "https://conundrumgirl.github.io/Synapse-React-Client/",
+  "homepage": "https://oceo-yj.github.io/Synapse-React-Client/",
```

Comment:
could you actually change this to "https://sage-bionetworks.github.io/Synapse-React-Client/"?